report.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+echo "Enter cluster name (default my-cluster)"
+read cluster
+if [ -z $cluster ]; then
+	cluster="my-cluster"
+fi
+
+get_yamls() {
+	mkdir -p ./reports/"$1"s
```

Comment:
You should use `mktemp -d` rather than using a hard-coded `reports` directory (you don't know for sure whether the user already has a `reports` directory, which you end up deleting!)